src/audio/kpb.c

```diff
@@ -1530,12 +1529,16 @@ static inline bool validate_host_params(struct comp_dev *dev,
 
 	if (!host_period_size || !host_buffer_size) {
 		/* Wrong host params */
+		comp_err(dev, "kpb: host_period_size (%d) cannot be 0 and host_buffer_size (%d) cannot be 0",
```

Comment:
Should we be using the new trace context API here ?

Reply:
This is the most recent one that I know. Other components still use it as well.